src/lib/select/select/select.ts

```diff
@@ -148,8 +148,6 @@ declare global {
  * @slot label - Renders its content as a positioned label.
```

Comment:
We can also remove the default/unnamed and `label` slots as well.

Additionally, this is missing the `value` slot that exists in only the select.